vim-engine/src/test/resources/engine_ex_commands.yaml

```diff
@@ -0,0 +1,129 @@
+# This file was automatically generated by [com.intellij.vim.processor.EXCommandProcessor].
```

Comment:
Why do we generate commands for test module?

Reply:
My bad, I thought you can't access  main resources from tests